internal/cas/errors.go

```diff
@@ -0,0 +1,82 @@
+package cas
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Error types that can be returned by the cas package
+type Error string
+
+func (e Error) Error() string {
+	return string(e)
+}
+
+const (
+	// ErrTempDir is returned when failing to create or close a temporary directory
+	ErrTempDir Error = "failed to create or manage temporary directory"
+	// ErrCreateDir is returned when failing to create a directory
+	ErrCreateDir Error = "failed to create directory"
+	// ErrHomeDir is returned when failing to find home directory
+	ErrHomeDir Error = "failed to find home directory"
```

Comment:
Looks like some of the constants aren't used

ErrHomeDir
ErrWriteToStore
ErrHardLink
ErrParseMode